File: tests/go/cases/case_187_dead_branch_global_09.go
package escape_tests

var retainedCase187 = []map[string]string{}

func Case187DeadBranchGlobal09(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_branch_global_09",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "dead_branch_global_09:" + raw,
	}
	if false {
		retainedCase187 = append(retainedCase187, payload)
	}
	// SAFE: dead branch retention.
	return payload["checksum"]
}

File: tests/rust/cases/case_187_dead_branch_global_09.rs
#![allow(unused)]

use std::collections::HashMap;
use std::sync::{Mutex, OnceLock};

#[allow(dead_code)]
static RETAINED_CACHE: OnceLock<Mutex<Vec<HashMap<String, String>>>> = OnceLock::new();
#[allow(dead_code)]
static RETAINED_AUDIT: OnceLock<Mutex<Vec<HashMap<String, String>>>> = OnceLock::new();
#[allow(dead_code)]
static RETAINED_HANDLERS: OnceLock<Mutex<Vec<String>>> = OnceLock::new();

pub fn case_187_dead_branch_global_09(input: String) -> String {
    let task_name = "dead_branch_global_09".to_string();
    let raw = if input.is_empty() { "sample".to_string() } else { input };
    let mut payload: HashMap<String, String> = HashMap::new();
    payload.insert("task".to_string(), task_name.clone());
    payload.insert("entity".to_string(), "stress".to_string());
    payload.insert("stage".to_string(), "evaluation".to_string());
    payload.insert("input".to_string(), raw.clone());
    payload.insert("checksum".to_string(), format!("{}:{}", task_name, raw.len()));
    if false {
        RETAINED_CACHE.get_or_init(|| Mutex::new(Vec::new())).lock().expect("cache lock").push(payload.clone());
    }
    // SAFE: dead branch retention cannot execute.
    payload.get("checksum").cloned().unwrap_or_default()
}
